src/Database/PostgreSQL/PQTypes/Model/Index.hs

```diff
@@ -89,7 +126,8 @@ sqlCreateIndex :: RawSQL () -> TableIndex -> RawSQL ()
 sqlCreateIndex tname idx@TableIndex{..} = mconcat [
     "CREATE "
   , if idxUnique then "UNIQUE " else ""
```

Comment:
@23Skidoo Only `BTree` indexes can ever be unique, so I guess we really should ignore any method supplied or throw. For the future?

Reply:
Shouldn't be too hard to add a check, I think.

Reply:
No, I just thought I was out of time but there's a weekend ahead so I'll probably do it then.